2024-11-02 13:30:56.379 [info] (Client) Pylance async client (2024.10.1) started with python extension (2024.18.0)
2024-11-02 13:30:56.504 [info] [Info  - 1:30:56 PM] (2857) Server root directory: file:///home/ec2-user/.vscode-server/extensions/ms-python.vscode-pylance-2024.10.1/dist
2024-11-02 13:30:56.509 [info] [Info  - 1:30:56 PM] (2857) Pylance language server 2024.10.1 (pyright version 1.1.382, commit e46efc6b) starting
2024-11-02 13:30:56.509 [info] [Info  - 1:30:56 PM] (2857) Starting service instance "ec2-user"
2024-11-02 13:30:56.638 [info] [Info  - 1:30:56 PM] (2857) Setting environmentName for service "ec2-user": "3.9.16 (global)"
2024-11-02 13:30:56.638 [info] [Info  - 1:30:56 PM] (2857) Setting pythonPath for service "ec2-user": "/bin/python3"
2024-11-02 13:30:56.639 [info] [Info  - 1:30:56 PM] (2857) No include entries specified; assuming /home/ec2-user
2024-11-02 13:30:56.640 [info] [Info  - 1:30:56 PM] (2857) Auto-excluding **/node_modules
2024-11-02 13:30:56.640 [info] [Info  - 1:30:56 PM] (2857) Auto-excluding **/__pycache__
2024-11-02 13:30:56.641 [info] [Info  - 1:30:56 PM] (2857) Auto-excluding **/.*
2024-11-02 13:30:56.715 [info] [Info  - 1:30:56 PM] (2857) Assuming Python version 3.9.16.final.0
2024-11-02 13:30:56.939 [info] [Info  - 1:30:56 PM] (2857) Found 2 source files
2024-11-02 13:30:58.836 [info] [Info  - 1:30:58 PM] (2857) Background analysis(1) root directory: file:///home/ec2-user/.vscode-server/extensions/ms-python.vscode-pylance-2024.10.1/dist
2024-11-02 13:30:58.855 [info] [Info  - 1:30:58 PM] (2857) Background analysis(1) started
2024-11-02 13:30:59.127 [info] [Info  - 1:30:59 PM] (2857) Indexer background runner(2) root directory: file:///home/ec2-user/.vscode-server/extensions/ms-python.vscode-pylance-2024.10.1/dist (index)
2024-11-02 13:30:59.135 [info] [Info  - 1:30:59 PM] (2857) Indexing(2) started
2024-11-02 13:31:01.443 [info] [Info  - 1:31:01 PM] (2857) [IDX(2)] Long operation: scan packages file:///home/ec2-user (2050ms)
2024-11-02 13:31:01.443 [info] [Info  - 1:31:01 PM] (2857) scanned(2) 87 files over 1 exec env
2024-11-02 13:31:02.124 [info] [Info  - 1:31:02 PM] (2857) indexed(2) 87 files over 1 exec env
2024-11-02 13:31:02.135 [info] [Info  - 1:31:02 PM] (2857) Indexing finished(2).
2024-11-02 13:59:36.700 [info] [Info  - 1:59:36 PM] (2857) Starting service instance "<default>"
2024-11-02 13:59:36.761 [info] [Info  - 1:59:36 PM] (2857) Setting environmentName for service "<default>": "3.9.16 (global)"
2024-11-02 13:59:36.766 [info] [Info  - 1:59:36 PM] (2857) Setting pythonPath for service "<default>": "/bin/python3"
2024-11-02 13:59:36.767 [info] [Info  - 1:59:36 PM] (2857) No include entries specified; assuming /<default workspace root>
2024-11-02 13:59:36.769 [info] [Info  - 1:59:36 PM] (2857) Auto-excluding **/node_modules
2024-11-02 13:59:36.769 [info] [Info  - 1:59:36 PM] (2857) Auto-excluding **/__pycache__
2024-11-02 13:59:36.774 [info] [Info  - 1:59:36 PM] (2857) Auto-excluding **/.*
2024-11-02 13:59:36.903 [info] [Info  - 1:59:36 PM] (2857) Assuming Python version 3.9.16.final.0
2024-11-02 13:59:37.109 [info] [Error - 1:59:37 PM] (2857) File or directory "/<default workspace root>" does not exist.
2024-11-02 13:59:37.113 [info] [Info  - 1:59:37 PM] (2857) No source files found.
2024-11-02 13:59:37.765 [info] [Info  - 1:59:37 PM] (2857) Background analysis(3) root directory: file:///home/ec2-user/.vscode-server/extensions/ms-python.vscode-pylance-2024.10.1/dist
2024-11-02 13:59:37.770 [info] [Info  - 1:59:37 PM] (2857) Background analysis(3) started
